bin/flutter

```diff
@@ -150,6 +150,13 @@ DART_SDK_PATH="$FLUTTER_ROOT/bin/cache/dart-sdk"
 DART="$DART_SDK_PATH/bin/dart"
 PUB="$DART_SDK_PATH/bin/pub"
 
+case "$(uname -s)" in
```

Comment:
Please add some comments explaining why this is here.

Reply:
Maybe something like this?

```suggestion
# If running over git-bash, overrides the default UNIX
# executables with win32 executables
case "$(uname -s)" in
```

Reply:
Yup, that sounds good. Thanks.